### ISSUE TITLE
Plotting feature request: (f, a, b) default behavior on f returning single val

### ISSUE THREAD

**Issue by tomerarnon** [JuliaDiffEq/DifferentialEquations.jl#361]
At the moment, e.g, `plot(sol, vars = [(f, 1, 2)])` requires that `f` return a tuple which will be plotted as `x, y`. For `f` returning a single value, plotting that value vs time (0th dim) would be a convenient behavior, just like is currently done for single values in e.g. `vars =[1, 2] <=> vars =[(0,1), (0,2)]`. 
Having this would mean that scalar operations such as (-, 1, 2) would be passable as is.

**Comment by ChrisRackauckas** [comment]
Yes, this is a very good idea. It shouldn't be too hard to do, just the tuple expansion code is a little tricky.
